crates/admin/src/cluster_controller/service.rs

```diff
@@ -322,8 +329,17 @@ impl<T: TransportConnect> Service<T> {
                     state.reconfigure(configuration);
                 }
                 result = state.run() => {
-                    let leader_event = result?;
-                    state.on_leader_event(&self.observed_cluster_state, leader_event).await?;
+                    let leader_event = match result {
+                        Ok(leader_event) => leader_event,
+                        Err(err) => {
+                            warn!("Failed to run cluster controller operations. This can impair the overall cluster operations: {:?}", err);
```

Comment:
```suggestion
                            warn!("Failed to run cluster controller operations. This can impair the overall cluster operations: {}", err);
```